plutus-ledger/src/Ledger/Scripts.hs

```diff
@@ -112,17 +126,17 @@ infrequently (I believe).
 -}
 instance Eq Script where
     {-# INLINABLE (==) #-}
-    a == b = BSL.toStrict (serialise a) == BSL.toStrict (serialise b)
+    a == b = flat a == flat b
```

Comment:
No need to change this, right? especially if `Script` keeps just its `Serialise` instance.

Reply:
Does this answer your question?
https://github.com/input-output-hk/plutus/pull/2482#issuecomment-731086632

Reply:
We should decide if we want to keep the CBOR instance for *Program*. We should *definitely* keep the CBOR instance for `Script`.